server/src/application/survey.ts

```diff
@@ -1479,14 +1488,24 @@ function dbSurveyToSurvey(
     },
     organization: dbSurvey.organization,
     tags: dbSurvey.tags,
+    enabledLanguages: dbSurvey.survey_languages,
   };
+
+  const enabledLanguages = dbSurvey.survey_languages.reduce(
+    (acc, lang) => {
```

Comment:
[nit] Not a big deal, especially in such small reducer functions, but I think the accumulator should be named to reflect the desired result, e.g. `languages` in this case. At least to me it'd look a bit cleaner that way.